src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -778,6 +780,14 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 else
                         arg_enabled = r;
 
+        } else if (proc_cmdline_key_streq(key, "systemd.gpt_auto.relax_var_check")) {
+
+                r = value ? parse_boolean(value) : 0;
+                if (r < 0)
+                        log_warning_errno(r, "Failed to parse gpt-auto relax-var-check switch \"%s\", ignoring: %m", value);
```

Comment:
I think it is better to clearly mention the full key in the log message:
`"gpt-auto relax-var-check"` -> `"systemd.gpt_auto.relax_var_check"`

Reply:
Done.